runtime/near-runtime-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# NEAR Runtime utilities
+
+This crate contains utility functions for NEAR runtime.
```

Comment:
We need to state somewhere that we don't want to pull dependencies that cannot be compiled in WASM or something along these lines, right? I would state it in this README and in the Cargo.toml in the `dependencies` section, so we don't accidentally forget and include `near-crypto` or `near-primitives`.